roles/gpu/tasks/nvdp.yml

```diff
@@ -25,6 +25,7 @@
       metadata:
         labels:
           allow-nvdp: "true"
+          nvidia.com/gpu.present: "true"
```

Comment:
As @Zblocker64 mentioned, this isn't needed as the inventory op assigns this label. (I am pointing at the exact label).